api/nakadi-event-bus-api.yaml

```diff
@@ -1591,18 +1591,18 @@ definitions:
 
           List of compatibility modes:
 
-          - 'fixed': Schema changes are not allowed. This is used only by legacy event types. This option is not
-            available for new event types. Schema validation is lenient and allow fields that are not defined. Also,
-            it's possible to use the full json schema specification for defining schemas.
-
-          - 'none': Any schema modification is accepted, even if it might break existing producers or consumers. When
-            validating events, no additional properties are accepted unless explicitly stated in the schema.
-
           - 'compatible': Consumers can reliably parse events produced under different versions. Every event published
             since the first version is still valid based on the newest schema. When in compatible mode, it's allowed to
             add new optional properties and definitions to an existing schema, but no other changes are allowed.
             Under this mode, the following json-schema attributes are not supported: `not`, `patternProperties`,
             `additionalProperties` and `additionalItems`. When validating events, additional properties is `false`.
+
+          - 'forward': Compatible schema changes are allowed. It's possible to use the full json schema specification
```

Comment:
Exactly. Also, I changed the default value to be `forward` and not `compatible`. This way we can release it in a backward compatible way.